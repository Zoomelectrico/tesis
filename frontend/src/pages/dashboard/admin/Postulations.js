import React from "react";
import {
  Container,
  Row,
  Col,
  Card,
  CardHeader,
  CardBody,
  Table
} from "reactstrap";
import { Header } from "../../../components";

const Postulations = props => (
  <>
    <Header />
    <Container className="mt--7">
      <Row>
        <Col sm="12">
          <Card>
            <CardHeader>
              <h2>Grupos Electorales - Postulaciones</h2>
            </CardHeader>
            <CardBody>
              <Table>
                <thead>
                  <tr>
                    <td>#</td>
                    <td>Grupo Electoral</td>
                    <td>Descargar</td>
                  </tr>
                </thead>
                <tbody>{/** Rows */}</tbody>
              </Table>
            </CardBody>
          </Card>
        </Col>
      </Row>
    </Container>
  </>
);

export default Postulations;
