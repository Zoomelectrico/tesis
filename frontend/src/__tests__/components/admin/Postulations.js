import React from 'react';
import { shallow } from 'enzyme';
import Postulations from '../../../pages/dashboard/admin/Postulations';

describe('Electoral Groups', () => {
  it('Render Correctly', () => {
    const wrapper = shallow(<Postulations />);
    expect(wrapper).toMatchSnapshot();
  });
});
