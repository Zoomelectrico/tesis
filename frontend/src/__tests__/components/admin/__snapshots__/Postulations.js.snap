// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Electoral Groups Render Correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Postulations />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <Header />,
        <Container
          className="mt--7"
          tag="div"
        >
          <Row
            tag="div"
          >
            <Col
              sm="12"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Card
                tag="div"
              >
                <CardHeader
                  tag="div"
                >
                  <h2>
                    Grupos Electorales - Postulaciones
                  </h2>
                </CardHeader>
                <CardBody
                  tag="div"
                >
                  <Loading />
                </CardBody>
              </Card>
            </Col>
          </Row>
        </Container>,
        <Toast />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Row
            tag="div"
          >
            <Col
              sm="12"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Card
                tag="div"
              >
                <CardHeader
                  tag="div"
                >
                  <h2>
                    Grupos Electorales - Postulaciones
                  </h2>
                </CardHeader>
                <CardBody
                  tag="div"
                >
                  <Loading />
                </CardBody>
              </Card>
            </Col>
          </Row>,
          "className": "mt--7",
          "tag": "div",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Col
              sm="12"
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <Card
                tag="div"
              >
                <CardHeader
                  tag="div"
                >
                  <h2>
                    Grupos Electorales - Postulaciones
                  </h2>
                </CardHeader>
                <CardBody
                  tag="div"
                >
                  <Loading />
                </CardBody>
              </Card>
            </Col>,
            "tag": "div",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Card
                tag="div"
              >
                <CardHeader
                  tag="div"
                >
                  <h2>
                    Grupos Electorales - Postulaciones
                  </h2>
                </CardHeader>
                <CardBody
                  tag="div"
                >
                  <Loading />
                </CardBody>
              </Card>,
              "sm": "12",
              "tag": "div",
              "widths": Array [
                "xs",
                "sm",
                "md",
                "lg",
                "xl",
              ],
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <CardHeader
                    tag="div"
                  >
                    <h2>
                      Grupos Electorales - Postulaciones
                    </h2>
                  </CardHeader>,
                  <CardBody
                    tag="div"
                  >
                    <Loading />
                  </CardBody>,
                ],
                "tag": "div",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": <h2>
                      Grupos Electorales - Postulaciones
                    </h2>,
                    "tag": "div",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Grupos Electorales - Postulaciones",
                    },
                    "ref": null,
                    "rendered": "Grupos Electorales - Postulaciones",
                    "type": "h2",
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": <Loading />,
                    "tag": "div",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <Header />,
          <Container
            className="mt--7"
            tag="div"
          >
            <Row
              tag="div"
            >
              <Col
                sm="12"
                tag="div"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
              >
                <Card
                  tag="div"
                >
                  <CardHeader
                    tag="div"
                  >
                    <h2>
                      Grupos Electorales - Postulaciones
                    </h2>
                  </CardHeader>
                  <CardBody
                    tag="div"
                  >
                    <Loading />
                  </CardBody>
                </Card>
              </Col>
            </Row>
          </Container>,
          <Toast />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Row
              tag="div"
            >
              <Col
                sm="12"
                tag="div"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
              >
                <Card
                  tag="div"
                >
                  <CardHeader
                    tag="div"
                  >
                    <h2>
                      Grupos Electorales - Postulaciones
                    </h2>
                  </CardHeader>
                  <CardBody
                    tag="div"
                  >
                    <Loading />
                  </CardBody>
                </Card>
              </Col>
            </Row>,
            "className": "mt--7",
            "tag": "div",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Col
                sm="12"
                tag="div"
                widths={
                  Array [
                    "xs",
                    "sm",
                    "md",
                    "lg",
                    "xl",
                  ]
                }
              >
                <Card
                  tag="div"
                >
                  <CardHeader
                    tag="div"
                  >
                    <h2>
                      Grupos Electorales - Postulaciones
                    </h2>
                  </CardHeader>
                  <CardBody
                    tag="div"
                  >
                    <Loading />
                  </CardBody>
                </Card>
              </Col>,
              "tag": "div",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <Card
                  tag="div"
                >
                  <CardHeader
                    tag="div"
                  >
                    <h2>
                      Grupos Electorales - Postulaciones
                    </h2>
                  </CardHeader>
                  <CardBody
                    tag="div"
                  >
                    <Loading />
                  </CardBody>
                </Card>,
                "sm": "12",
                "tag": "div",
                "widths": Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ],
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": Array [
                    <CardHeader
                      tag="div"
                    >
                      <h2>
                        Grupos Electorales - Postulaciones
                      </h2>
                    </CardHeader>,
                    <CardBody
                      tag="div"
                    >
                      <Loading />
                    </CardBody>,
                  ],
                  "tag": "div",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "children": <h2>
                        Grupos Electorales - Postulaciones
                      </h2>,
                      "tag": "div",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Grupos Electorales - Postulaciones",
                      },
                      "ref": null,
                      "rendered": "Grupos Electorales - Postulaciones",
                      "type": "h2",
                    },
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "children": <Loading />,
                      "tag": "div",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "function",
                      "props": Object {},
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
